deptrack-devops-recipes/src/Devops/Bootstrap/Parasite.hs

```diff
@@ -184,3 +184,34 @@ remotedWith mkAction usr clo host =
                                 , "-l", unpack (userName u), unpack ip
                                 , remoteExecution rpath b64 ]
         remoteExecution rpath b64 = unwords [ "chmod", "+x", rpath, ";", rpath, magicRemoteArgv, unpack b64 , ";"]
+
+
+-- | Remotely callback some `Continued a` on some parasited host.
+--
+remoteContinued :: (Typeable a)
+               => DevOp User -> Continued a -> DevOp ParasitedHost -> DevOp (Remoted a)
+remoteContinued usr cont host =
+  devop fst mkOp (do
+              u <- usr
+              c <- ssh
+              let obj = eval cont
+                  (BinaryCall _ fArgs) = callback cont
+                  args = fArgs TurnUp
+              (ParasitedHost rpath _ r) <- host
+              return (Remoted r obj,(rpath, c, u,args,  r)))
+
+  where ssh = binary :: DevOp (Binary "ssh")
+        mkOp (_, (rpath, c, u, args, r)) = buildOp
+              ("remote-continued: " <> " @" <> pack (show $ resolvedKey r))
+              ("calls 'turnup' at host " <> pack (show $ resolvedKey r))
+              noCheck
+              (do
+                  Remote ip <- resolver r
+                  blindRun c (sshCmd rpath u ip args) "")
+              noAction
```

Comment:
That's more an open question than a comment --sorry-- : I'm wondering if we should propagate the TurnDown/Reload operations as well. A part of me says "no" because we should encourage obliterating remote VMs all at once, another part of me says "yes" for cleanliness-of-the-approach purposes. Just putting this in your brain now and we can see with more datapoints as we get more mileage :-).